docs/src/main/asciidoc/infinispan-client.adoc

```diff
@@ -442,6 +442,43 @@ for Kubernetes deployments, Infinispan Console,
 https://infinispan.org/docs/stable/titles/rest/rest.html#rest_v2_protobuf_schemas[REST API] or the
 https://infinispan.org/docs/stable/titles/encoding/encoding.html#registering-sci-remote-caches_marshalling[Hot Rod Java Client].
 
+[#annotations-api]
```

Comment:
I don't see the fully qualified name anywhere in this section. I am pretty sure that will lead users to think that they can use the regular Quarkus caching annotations